lib/table.js

```diff
@@ -301,10 +306,7 @@ Table.prototype.update = function (item, options, callback) {
 
   const schemaValidation = internals.validateItemFragment(item, self.schema);
   if (schemaValidation.error) {
-    return callback(_.assign(new Error(`Schema validation error while updating item in table ${self.tableName()}: ${JSON.stringify(schemaValidation.error)}`), {
-      name: 'DynogelsUpdateError',
-      detail: schemaValidation.error
-    }));
+    return callback(schemaValidation.error);
```

Comment:
Why aren't you calling back with the previous Error object that you created? Maybe I'm being dense but we would want to right?